src/main/resources/XWiki/SuggestDisplay.xml

```diff
@@ -258,13 +264,52 @@
     img.observe('click', this.removeItem.bindAsEventListener(this));
     li.appendChild(span);
     li.appendChild(hiddenInput);
+    //Add the edit button
+    if(this.inEditItemsMode($(this.list).up(0))){
```

Comment:
The role of this method is to check if the 'edit items mode' is activated in the displayer-multiselect-suggest.

This method check if there is a hidden input that have a 'inEditItemsMode' css class in the html container that inclose the display-multiselect-suggest html code.

The inEditItemsMode method have a parameter (elem), this parameter is a DOM element that (if the edit mode is activated) contains the hidden input that indicates that the edit mode is activated.

The elem parameter is changing depending on the location where the method is called in the code.

I think it should be more clear, I will try to improve it.